packages/core/src/App/AppContent.tsx

```diff
@@ -47,24 +46,19 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
         switchLanguage(current_language);
     }, [current_language, switchLanguage]);
 
+    React.useEffect(() => {
+        if (isPhoneNumberVerificationGBLoaded && isPhoneNumberVerificationEnabled) {
+            store.client.setIsPhoneNumberVerificationEnabled(isPhoneNumberVerificationEnabled);
+        }
+    }, [isPhoneNumberVerificationEnabled, store.client, isPhoneNumberVerificationGBLoaded]);
```

Comment:
should we add store.client.setIsPhoneNumberVerificationEnabled as the dependency?

Reply:
sure let me update it 👌